packages/core/misc-widgets/src/locales/en.json

```diff
@@ -0,0 +1,6 @@
+{
+  "header": {
+    "title": "Title of the header",
+    "content": "Content of the header"
+  }
+}
```

Comment:
You can delete the locales folder since it's not currently needed.

Reply:
Actually just added i18n so keeping it now.